packages/fxa-settings/src/components/ConnectedServices/index.tsx

```diff
@@ -0,0 +1,112 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useState } from 'react';
+import { AlertBar } from '../AlertBar';
+import { ButtonIconReload } from '../ButtonIcon';
+import { useAlertBar } from 'fxa-settings/src/lib/hooks';
+import { useAccount, useLazyAccount } from '../../models';
+import { LinkExternal } from 'fxa-react/components/LinkExternal';
+
+const UTM_PARAMS =
+  '?utm_source=accounts.firefox.com&utm_medium=referral&utm_campaign=fxa-devices';
+const DEVICES_SUPPORT_URL =
+  'https://support.mozilla.org/kb/fxa-managing-devices' + UTM_PARAMS;
+
+export const Service = () => {
+  return (
+    <section
+      className="my-1"
+      id="service"
+      data-testid="settings-connected-service"
+    >
+      <div className="border-2 border-solid border-grey-100 rounded flex justify-around items-center">
+        <span>icon</span>
+
+        <div className="flex flex-col">
+          <h2>Service Name</h2>
+          <p className="text-sm">location</p>
+        </div>
+
+        <p>Active Now</p>
+
+        <button
+          className="cta-neutral cta-base disabled:cursor-wait whitespace-no-wrap"
+          data-testid="connected-service-sign-out"
+        >
+          Sign out
+        </button>
+      </div>
+    </section>
+  );
+};
+
+export const ConnectedServices = () => {
+  const alertBar = useAlertBar();
+  const [errorText, setErrorText] = useState<string>();
+  const onError = (e: Error) => {
+    setErrorText(e.message);
+    alertBar.show();
+  };
+  const [getAccount, { accountLoading }] = useLazyAccount((error) => {
+    setErrorText('Sorry, there was a problem refreshing the recovery key.');
+    alertBar.show();
+  });
+
+  return (
+    <section
+      className="mt-3"
+      id="connected-services"
+      data-testid="settings-connected-services"
+    >
+      <h2 className="font-header font-bold ml-4 mb-4">Connected Services</h2>
+      <div className="bg-white tablet:rounded-xl shadow py-4 px-6">
```

Comment:
I'm seeing `27px` of space between "Everything you are using and signed into" and the top of the section and `37px` at the bottom, so `py-4` should be `pt-7 pb-9` to get pretty close, and then we can take off `mb-4` on the container div of "Missing or duplicate items?"